app/sample/gpu_direct/rx_st20_pipeline_gpu_direct.c

```diff
@@ -0,0 +1,259 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ */
+
+#include "../sample_util.h"
+
+struct rx_st20p_sample_ctx {
```

Comment:
It'd be really great if you add a one-line description to every struct and function you add